### PR TITLE
Feature/landing reformat

### PR DESCRIPTION
**Pull request by frapperino**

### CODE REVIEW

File: src/sections/landing/media-logos.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import Img from 'gatsby-image';
+
+interface Props {
```
Comment (hedvigoscar) [inline src/sections/landing/media-logos.tsx]: I think we can probably type this better right now 🙂

File: src/sections/landing/media-logos.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  image: any; // TODO type this better?
+  sizes: object; // TODO type this better?
+}
+
+const MediaLogos: React.SFC<Props> = (props) => (
+  <div className="Container">
```
Comment (hedvigoscar) [inline src/sections/landing/media-logos.tsx]: We can probably improve upon the concept here by only taking `image` in props and not even rendering the `<div>` if we lack an image (and instead return `null`)

File: src/sections/landing/media-logos.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    {props.image && <Img className="Home-media" sizes={props.sizes} alt="" />}
```
Comment (hedvigoscar) [inline src/sections/landing/media-logos.tsx]: Given that `Home-media` only occurs once we can improve this by making it a `styled(Img)` and remove the old CSS 🙂

File: src/templates/student-page.js [hunk 4]
```diff
@@ -173,7 +173,7 @@ class StudentTemplate extends React.Component {
               <div className="Container u-hidden u-lg-block">
                 {mediaLogosFile && (
                   <Img
-                    className="Home-media"
+                    className="Home-media" // TODO: use emotion and remove css when extracted to section
```
Comment (palmfjord) [inline src/templates/student-page.js]: how come we can't reuse the component we just created?

Comment (hedvigoscar) [reply src/templates/student-page.js]: I guess I can answer: The old behaviour (contained in the css classes) where media logos are on top or bottom depending on device size remians on student page.

If this behaviour is deemed undesirable we can reuse the component we created.

Comment (palmfjord) [reply src/templates/student-page.js]: right

Comment (frapperino) [reply src/templates/student-page.js]: The old behaviour on the student page actually removes the media logos below width at 960px. I think both pages should be consistent, so I opt for using the same component

**Pull request merged**